compute/admin_guide/agentless-scanning/agentless-scanning.adoc

```diff
@@ -92,3 +92,105 @@ Failed cleaning up network resources: UnauthorizedOperation: You are not authori
+=== Scan Progress and Statuses
+
+Accounts go through the following statuses as a scan takes place.
+
+. *New* - if an account was added during an ongoing scan or between scan cycles.
+
+. *Pending Scan* - all accounts are set to this state when a scan cycle begins.
+
+. *Scanning* - the scan process takes place, including pre-flight checks for permissions.
+Hosts that were done scanning would immediately appear in the scan results - results are never delayed to the end of a scan cycle.
+At this state, after an account has finished scanning, all scanners, snapshots and disks are being cleaned up immediately.
+
+. *Pending Cleanup* - final state for all accounts that finished scanning. Accounts move to cleanup only after all accounts have reached this state.
+
+. *Cleanup* - if networking resources were created, they are cleaned up across all accounts and regions.
+
+. *Completed* - scanning and cleanup is completed, this could be one of two options:
+
+.. Completed successfully.
+
+.. Completed with errors if issues occurred during any stages of the scan.
+
+[#cycle-length]
+=== Scan Cycle Length
+
+It is nearly impossible to estimate what is the end-to-end duration of a scan cycle since it depends on a variety of factors, for example.
+
+* Number of hosts
+* Hosts disks sizes
+* Used space on hosts disks
+* Number of files in the hosts disks
+* How the hosts are dispersed between accounts and regions, for example: a single account with a single region that contains 100 hosts, is scanned faster than 10 accounts with 10 regions each, that contains a single host in every region.
+
+* CSP-related factors:
+
+** API calls latency
+** API calls errors
+
+[#snapshots-creation]
+=== Snapshots Creation
+
+During the agentless scanning process, Prisma Cloud iterates through all regions within your environment and creates a snapshot of each host in every region.
+To mitigate the security risk that non-running hosts pose,you can enable agentless scanning and scan non-running hosts by configuring the agentless scanning for every account you onboard.
+Each scanner instance is attached with a maximum of 26* snapshots, which it then scans for security risks.
+
+By default, agentless scanning is configured to spin up a single agentless scanner within every region, meaning that at any given time, only a single agentless scanner is deployed in every region.
+The agentless scanner scans hosts snapshots iteratively within every region in batches of 26 snapshots at a time.
+
+[#scaling-agentless-scanning]
+=== Scaling and Parallel Scanning
+
+To expedite the scanning process, you can enable auto scaling under the agentless configuration, this enables the scanning process to spin up to 50 agentless scanners in parallel to scan the snapshots within every region. If enabled, auto-scaling allows scanning of up to 50*26=1300 hosts in parallel within every region.
+For OCI accounts, the maximum snapshots scanned per agentless scanner is set to 16 snapshots.
```

Comment:
this should have an asterisk sign * at the beginning since it is referenced in the text above

Reply:
@omri86 That's not how referencing works in our content because it's ambiguous.  I can do one of the following:

- Move this line further up for it to be closer to where we mention the snapshots.
- Reiterate here the number of snapshots per scanner being 26.
- Make it a note in the Snapshots section.